app/src/main/res/layout/view_session_cell.xml

```diff
@@ -32,22 +26,24 @@
 
         <View
                 android:id="@+id/categoryBorder"
-                android:layout_width="match_parent"
+                android:layout_width="0dp"
                 android:layout_height="3dp"
-                android:layout_marginEnd="-6dp"
-                android:layout_marginLeft="-6dp"
-                android:layout_marginRight="-6dp"
-                android:layout_marginStart="-6dp"
```

Comment:
ConstraintLayout doesn't support negative margin. We should add positive margin to other views.